src/Sylius/Behat/Context/Setup/ZoneContext.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Setup;
+
+use Behat\Behat\Context\Context;
+use Sylius\Bundle\AddressingBundle\Factory\ZoneFactoryInterface;
+use Sylius\Bundle\SettingsBundle\Manager\SettingsManagerInterface;
+use Sylius\Component\Addressing\Model\ZoneInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ZoneContext implements Context
+{
+    /**
+     * @var array
+     */
+    private $euMembers = array(
+        'BE', 'BG', 'CZ', 'DK', 'DE', 'EE', 'IE', 'GR', 'ES',
+        'FR', 'IT', 'CY', 'LV', 'LT', 'LU', 'HU', 'MT', 'NL',
+        'AT', 'PL', 'PT', 'RO', 'SI', 'SK', 'FI', 'SE', 'GB',
+    );
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $zoneRepository;
+
+    /**
+     * @var SettingsManagerInterface
+     */
+    private $settingsManager;
+
+    /**
+     * @var ZoneFactoryInterface
+     */
+    private $zoneFactory;
+
+    /**
+     * @param RepositoryInterface $zoneRepository
+     * @param SettingsManagerInterface $settingsManager
+     * @param ZoneFactoryInterface $zoneFactory
+     */
+    public function __construct(RepositoryInterface $zoneRepository, SettingsManagerInterface $settingsManager, ZoneFactoryInterface $zoneFactory)
+    {
+        $this->zoneRepository = $zoneRepository;
+        $this->settingsManager = $settingsManager;
+        $this->zoneFactory = $zoneFactory;
+    }
+
+    /**
+     * @Transform /^"([^"]+)" zone$/
+     * @Transform /^zone "([^"]+)"$/
+     */
+    public function castZoneCodeToZone($zoneCode)
+    {
+        if (null === $zone = $this->zoneRepository->findOneBy(array('code' => $zoneCode))) {
+            throw new \Exception('Zone with code "'.$zoneCode.'" does not exist');
+        }
+
+        return $zone;
+    }
+
+    /**
+     * @Given /^there is "EU" zone containing all members of European Union$/
+     */
+    public function thereIsEUZoneContainingAllMembersOfEuropeanUnion()
+    {
+        $zone = $this->zoneFactory->createWithMembers($this->euMembers);
+        $zone->setType(ZoneInterface::TYPE_COUNTRY);
+        $zone->setCode('EU');
+        $zone->setName('European Union');
+
+        $this->zoneRepository->add($zone);
+    }
+
+    /**
+     * @Given /^default zone is "([^"]+)"$/
+     */
+    public function defaultZoneIs($zoneCode)
+    {
+        if (null === $zone = $this->zoneRepository->findOneBy(array('code' => $zoneCode))) {
+            throw new \Exception('Zone with code "'.$zoneCode.'" does not exist');
+        }
```

Comment:
Also, this step is wrong, we do not have a concept of default zone, we must be more specific: ``Default tax zone is ...```